x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -22,6 +22,8 @@ export const DEFAULT_DATE_FORMAT_TZ = 'dateFormat:tz';
 export const DEFAULT_DARK_MODE = 'theme:darkMode';
 export const DEFAULT_INDEX_KEY = 'securitySolution:defaultIndex';
 export const DEFAULT_NUMBER_FORMAT = 'format:number:defaultPattern';
+export const DEFAULT_INDEX_PATTERN_ID = 'security-solution';
```

Comment:
Just curious, is there some standard pattern for these across Kibana? I.e. camel-case, snake-case, etc.. ?

Reply:
no, the id is generated and looks like `a855f700-f5f9-11eb-8763-01917c58cbf9` so i followed that. i think `-` is standard for URL which is where these are stored